locales/en.json

```diff
@@ -37,6 +37,7 @@
     "keys_removed": "You lost keys to the vehicle!",
     "no_keys": "You don't have keys to this vehicle.",
     "not_near": "There is nobody nearby to hand keys to",
+    "vehicle_not_near": "Not close enough to vehicle",
```

Comment:
Should this be added to all other locale files? So that they at least have something default in English.

Reply:
done.